src/components/MessageLogger/Counter.module.scss

```diff
@@ -0,0 +1,12 @@
+#count {
+  background-color: #cfcece;
+  display: flex;
+  align-items: center;
+  border-radius: 40%;
```

Comment:
If the element is wide, a percentage border-radius looks like a streched circle. Use rem instead